.github/workflows/checks.yaml

```diff
@@ -98,6 +101,10 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ github.ref_name }}-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: |
             ${{ runner.os }}-cargo-${{ github.ref_name }}
+      - name: Cleanup Redis
```

Comment:
**suggestion (non-blocking):** 
please add 
`if: ${{ always() }}`

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always